src/core/server/saved_objects/migrations/core/elastic_index.ts

```diff
@@ -51,6 +52,9 @@ export async function fetchInfo(callCluster: CallCluster, index: string): Promis
       aliases: {},
       exists: false,
       indexName: index,
+      settings: {
+        'indices.id_field_data.enabled': true,
```

Comment:
I am not sure what this would do here. In addition, this is a cluster-level setting, so this won't work. I believe the solution here is to remove any sort/aggregating we have on `_id`